instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -71,7 +72,17 @@ def get_excluded_urls():
     return ExcludeList(urls)
 
 
+def get_traced_request_attrs():
+    attrs = configuration.Configuration().TORNADO_TRACED_REQUEST_ATTRS or ""
+    if attrs:
+        attrs = [attr.strip() for attr in attrs.split(",")]
+    else:
+        attrs = []
+    return attrs
+
+
 _excluded_urls = get_excluded_urls()
+_traced_attrs = get_traced_request_attrs()
```

Comment:
What about this to not have the function above?

```suggestion
_traced_attrs = [attr.strip() for attr in configuration.Configuration().TORNADO_TRACED_REQUEST_ATTRS.split(",")] if configuration.Configuration().TORNADO_TRACED_REQUEST_ATTRS else []
```

Reply:
Very similar functionality above uses a function so I think it's better to be consistent with existing code.